packages/conf-zig/conf-zig.1/opam

```diff
@@ -17,6 +17,7 @@ depexts: [
   ["zig"] {os = "freebsd"}
   ["zig"] {os = "macos" & os-distribution = "homebrew"}
   ["system:zig"] {os = "win32" & os-distribution = "cygwinports"}
+  ["zig"] {os-distribution = "cygwin"}
 ]
```

Comment:
```suggestion
]
depends: [ "conf-pkg-config" ]
```

due to 
```
#=== ERROR while compiling conf-zlib.1 ========================================#
"pkg-config": command not found.
```

Reply:
The error was in conf-zlib.1, not conf-zig.1 (but see note below)